### PR TITLE
Run virtual iridium before tests

### PR DESCRIPTION
**Pull request by hhenry01**
### Description

- Start a virtual iridium server whenever test.sh is ran and the src/network_systems folder exists
- Needed for https://github.com/UBCSailbot/network_systems/pull/52 and later local/remote transceiver and integration tests

### Verification

- [ ] 

### Resources

-

### CODE REVIEW

**Review by patrick-5546 — APPROVED** [review]
I'll approve so that once you fix CI it will merge.

At the end of colcon-test:
image
https://github.com/UBCSailbot/sailbot_workspace/actions/runs/6741485392/job/18326157764?pr=184#step:4:498

**Comment by hhenry01** [comment]
> I'll approve so that once you fix CI it will merge.
> 
> At the end of colcon-test: image https://github.com/UBCSailbot/sailbot_workspace/actions/runs/6741485392/job/18326157764?pr=184#step:4:498

Yeah socat exits with 143 for some reason. I'll need to figure out a way to either make it exit 0 or just suppress it.

**Pull request merged**
